Makefile

```diff
@@ -109,7 +109,6 @@ install: | prepare
 	@if [ $(ENV) = "development" ]; then \
 		$(MAKE) -s drush en $(DEVELOPMENT_MODULES); \
 	fi
-	$(MAKE) -s drush en $(shell ls -m modules/features)
```

Comment:
We can enable custom features and modules which disabled.
List of modules will define in env.default

Reply:
I didn't see it being defined in .env.default - looks like you changed your mind :) Or is it smth for the future?

Reply:
^^ if we need =) I mean, not default modules will not installed. But maybe we need enable all our modules in dev env.